src/ansi.rs

```diff
@@ -119,7 +119,7 @@ impl Mul<f32> for Rgb {
             b: (f32::from(self.b) * rhs).clamp(0.0, 255.0) as u8,
         };
 
-        trace!("Scaling RGB by {} from {:?} to {:?}", rhs, self, result);
+        log::trace!("Scaling RGB by {} from {:?} to {:?}", rhs, self, result);
```

Comment:
This change confuses me. What's the motivation behind it?

Reply:
To remove `cfg` because `trace` is not used under some feature.